### PR TITLE
Jobs related fixes

### PR DESCRIPTION
**Pull request by Garen7**
## About The Pull Request
<details>
<summary>
	This started as just a fix to the sechud but I saw some other bugs that were easy to fix so I fixed those too in the same PR, if you'd rather me break it up into multiple PRs thats fine just ask because I really didn't mean to have all that stuff in this one PR I just did it by accident.
</summary>
<hr>

	
<hr>
</details>

## Changelog
:cl: Garen7
tweak: tweaked the colors on the preachers sec hud icon to properly reflect the fact that they are a head of staff
tweak: the preacher will be treated like the other heads now in that they will get a part of the nuke code, a heavier poll weight, appear under command in the crew manifest, etc.
fix: NeoTheology members will once again appear on the crew manifest
fix: Vagabonds appear on the sechud again
fix: The First Officer can reassign people into NeoTheology and Club staff now
imageadd: Added sechud icons for the NeoTheology and Club staff
spellcheck: Replaced some instances of "Neotheology" with "NeoTheology"
admin: admins can job ban people from NeoTheology jobs like how they can with any other job
/:cl:

image
image

>tweak: tweaked the colors on the preachers sec hud icon to properly reflect the fact that they are a head of staff
>imageadd: Added sechud icons for the NeoTheology and Club staff
>fix: Vagabonds appear on the sechud again

Are all entirely just changes to hud.dmi

Also all these are tested and work

### CODE REVIEW

**Comment by GrayRachnid** [comment]
Also can you double check to make sure the Preacher appears as command on the manifest now.

File: code/game/jobs/job/church.dm [hunk 1]
```diff
@@ -1,8 +1,9 @@
 /datum/job/chaplain
 	title = "NeoTheology Preacher"
 	flag = CHAPLAIN
+	head_position = 1
 	department = DEPARTMENT_CHURCH
-	department_flag = CHURCH
+	department_flag = CHURCH | COMMAND
```
Comment (Garen7) [inline code/game/jobs/job/church.dm]: @GrayRachnid yup I already tested to make sure the preacher appears under command on the manifest, this is the line that causes that.

**Comment by Garen7** [comment]
@GrayRachnid sure ill swap those icons in but you sent the acolyte icon twice, can you send me the agrolyte icon when you get a chance?

**Comment by GrayRachnid** [comment]
https://cdn.discordapp.com/attachments/630282499997564940/630293933842366484/hudneotheologyagrolyte.dmi

**Comment by mattroks101** [comment]
This ready for review yet?

**Pull request merged**